segments/input/bpf/packet_dumper.go

```diff
@@ -0,0 +1,289 @@
+//go:build linux
+// +build linux
+
+// based on https://github.com/bwNetFlow/bpf_flowexport/blob/master/packetdump/packetdump.go
```

Comment:
Same here I guess? The reason we initially kept this outside the repo was that it stays separate and we can leverage any other kind of bpf packet capture. At this point, there should be lots of projects out there that we could use instead of our home-baked solution (I found `go-pkt` just now for instance)